107/src/main/java/org/ehcache/jsr107/Eh107CacheManager.java

```diff
@@ -215,6 +220,8 @@ public Properties getProperties() {
       onHeapStoreServiceConfig.storeByValue(jsr107Config.isStoreByValue());
     }
 
+    builder.addServiceConfig(new EhcacheServiceConfiguration().jsr107CompliantAtomics(ehcacheService.jsr107CompliantAtomics()));
```

Comment:
This is redundant with the change introduced in `EhcacheManager` that takes care of it also. And following that, passing the additional constructor argument is redundant too.